packages/es-components/src/components/controls/radio-buttons/RadioGroup.js

```diff
@@ -33,16 +33,12 @@ function RadioGroup({
   onChange,
   introContent,
   validationState,
-  additionalHelpContent,
-  theme
+  additionalHelpContent
 }) {
+  const theme = useTheme();
```

Comment:
`RadioButton` itself is using this `useTheme` hook, so there isn't a need to pass this as a prop.